app/src/main/java/se/inera/intyg/minaintyg/logging/MonitoringLogServiceImpl.java

```diff
@@ -37,10 +39,20 @@ public void logCertificateSent(String certificateId, String type, String recipie
     logEvent(MonitoringEvent.CERTIFICATE_SEND, certificateId, type, recipient);
   }
 
+  @Override
+  public void logClientError(String id, String code, String message, String stackTrace) {
+    logError(MonitoringEvent.CLIENT_ERROR, id, code, message,
+        Strings.isNullOrEmpty(stackTrace) ? NO_STACK_TRACE : stackTrace);
+  }
+
   private void logEvent(MonitoringEvent event, Object... logMsgArgs) {
     log.info(LogMarkers.MONITORING, buildMessage(event), logMsgArgs);
   }
 
+  private void logError(MonitoringEvent event, Object... logMsgArgs) {
+    log.error(LogMarkers.MONITORING, buildMessage(event), logMsgArgs);
```

Comment:
In Webcert & Rehabstöd this is logged on info level. Are we sure we want to log error here instead of info?